src/components/Header/index.js

```diff
@@ -1,18 +1,35 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
 
+import { Nav, Link } from './styled'
 import Urls from '../../constants/urls'
 
+const Links = [
+  {
+    path: Urls.index,
+    label: 'Homepage',
+  },
+  {
+    path: Urls.signUp,
+    label: 'Sign Up',
+  },
+  {
+    path: Urls.signIn,
+    label: 'Sign In',
+  },
+  {
+    path: Urls.cart,
+    label: 'Cart',
+  },
+]
+
 const Header = () => (
-  <nav>
-    <Link to={Urls.index}>Homepage</Link>
-    <span> |</span>
-    <Link to={Urls.signUp}>Sign Up</Link>
-    <span> |</span>
-    <Link to={Urls.signIn}>Sign In</Link>
-    <span> |</span>
-    <Link to={Urls.cart}>Cart</Link>
-  </nav>
+  <Nav>
```

Comment:
Responsivity of Navigation on mobile devices is missing

Reply:
![iPhone 5_SE](https://user-images.githubusercontent.com/12123231/55393819-3fe7cd00-553e-11e9-98f0-ee0a1633fcd3.png)

Reply:
Actually i dont care about responsive version. I have some troubles with programming logic, so i spend less time with css. 

FIXED!